alembic.ini

```diff
@@ -0,0 +1,73 @@
+# Uber specific alembic configuration
+
+[alembic]
+# The script_location is derived from the uber configuration, so this value is
+# completely ignored.
+# IGNORED
+script_location = alembic
```

Comment:
Though this value is ignored, it must not be commented out or the `alembic.config.Config` constructor will complain.